trunk/SUAVE/Plots/Performance/Mission_Plots.py

```diff
@@ -45,7 +45,7 @@ def plot_altitude_sfc_weight(results, line_color = 'bo-', save_figure = False, s
     """	  
     axis_font = {'size':'14'} 
     fig = plt.figure(save_filename)
-    fig.set_size_inches(10, 8) 
+    fig.set_size_inches(8, 5) 
```

Comment:
rather than hard code all of these and change all of them, default the size

Reply:
updated and defaulted